registry/base_configuration_listener.go

```diff
@@ -100,7 +100,13 @@ func ToConfigurators(urls []*common.URL, f func(url *common.URL) config_center.C
 			configurators = []config_center.Configurator{}
 			break
 		}
-		//TODO:anyhost_key judage
+
+		override := url.GetParams()
+		delete(override, constant.ANYHOST_KEY)
+		if len(override) == 0 {
+			configurators = nil
```

Comment:
It seems will clear `configurators` when url.GetParams()  is nil. Is it right logic?
Like:

Fisrt loop: url.GetParams() is not null, add a element to `configurators`.
Second loop :url.GetParams() is null ,will clear `configurators`?

Reply:
I am also puzzled by the java implememt that I follow. Could some one help to check it?

Reply:
> It seems will clear configurators when url.GetParams() is nil. Is it right logic?

I think `configurators  = nil` should be removed.

Reply:
> > It seems will clear configurators when url.GetParams() is nil. Is it right logic?
> 
> I think `configurators = nil` should be removed.

Wilson will fix it.